vagrant_config.json

```diff
@@ -208,7 +208,8 @@
       },
       "ruby":{
         "enabled":true,
-        "version": "1.9.3-p194",
+        "versions": ["1.9.3-p551", "2.2.3"],
+        "global_version": "1.9.3-p551",
```

Comment:
let's use 2.2.3 as global version instead.

I'll check if 1.9.3-p551 or 1.9.3-p194 is required anymore (for dev, blog, etc), if not, we could remove it.